CONTRIBUTION.md

```diff
@@ -1,15 +1,50 @@
-# We love contributions to figma/primer
+# We love contributions
 
-If you find a bug or issue, feel free to leave a comment in Figma and mention @ashygee.
+We're thrilled that you want to contribute to this primer/figma. Your help is essential for keeping it great.
```

Comment:
Question - are we inviting open source contributions at this time?

Reply:
Technically if they are bugfixes, etc. yes. I don't see a reason why not. However Primer will of course be favoring github so we may decide stuff because it makes sense in figma.